lib/src/domain/usecase/load_cast_to_movie_usecase.dart

```diff
@@ -0,0 +1,30 @@
+import 'package:http/http.dart';
+
+import '../../core/usecase/i_usecase.dart';
+import '../../core/util/data_state.dart';
+import '../../core/util/enums.dart';
+import '../../data/repository/api_repository.dart';
+import '../entity/cast.dart';
+import '../entity/movie.dart';
+import '../repository/i_my_repository.dart';
+
+class LoadCastToMovieUseCase extends IUseCase<Future<DataState<List<Cast>>>, Movie> {
```

Comment:
Remove use case logic

Reply:
Okay, same as the comment above, i'll have to ckeck anyway if the params are null or not

Reply:
Fixed